spec/requests/user/user_settings_spec.rb

```diff
@@ -55,6 +55,20 @@
         get "/settings/account"
         expect(response.body).to include ghost_account_message
       end
+
+      it "renders CONNECT_WITH_TWITTER and user with only github identity" do
+        user.identities.where(provider: "twitter").delete_all
+        get "/settings"
+        expect(response.body).to include "CONNECT TWITTER ACCOUNT"
+      end
+
+      it "renders does not render CONNECT_WITH_TWITTER if SiteConfig does not include Twitter auth" do
+        user.identities.where(provider: "twitter").destroy_all
+        SiteConfig.authentication_providers = ["github"]
```

Comment:
This is going to change the value for all subsequent tests though if not run in the correct order, we might want to store the previous value and restore it at the end of the test, or mock it if possible

Reply:
👍

Reply:
Consider using ClimateControl (https://github.com/thoughtbot/climate_control) (or a more modern/updated alternative) to set env variables for these tests!